doc/TESTING.md

```diff
@@ -66,21 +94,3 @@ To update to the latest unstable build of Nicotine+, run the following:
 cd nicotine-plus
 git pull
 ```
```

Comment:
_Add at bottom of Source section (last line of document):_

For information about Nicotine+ development procedures for maintainers, developers and code contributors, see [DEVELOPING.md](DEVELOPING.md).